src/Compilers/Test/Utilities/CSharp/NullableAttributesVisitor.cs

```diff
@@ -46,19 +50,40 @@ public override void VisitNamespace(NamespaceSymbol @namespace)
 
         public override void VisitNamedType(NamedTypeSymbol type)
```

Comment:
>VisitNamedType [](start = 29, length = 14)

Do we check attributes on implemented interfaces?

Reply:
Not visiting type references.

---
In reply to: [296931569](https://github.com/dotnet/roslyn/pull/36152#discussion_r296931569) [](ancestors = 296931569)